scripts/install_bx.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Download Bluemix CLI"
+wget --quiet --output-document=/tmp/Bluemix_CLI_amd64.tar.gz  http://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/latest/Bluemix_CLI_amd64.tar.gz
+tar -xf /tmp/Bluemix_CLI_amd64.tar.gz --directory=/tmp
+
+# Create bx alias
+echo "#!/bin/sh" >/tmp/Bluemix_CLI/bin/bx
+echo "/tmp/Bluemix_CLI/bin/bluemix \"\$@\" " >>/tmp/Bluemix_CLI/bin/bx
+chmod +x /tmp/Bluemix_CLI/bin/*
+
+export PATH="/tmp/Bluemix_CLI/bin:$PATH"
+
+# Install Armada CS plugin
+echo "Install the Bluemix container-service plugin"
+bx plugin install container-service -r Bluemix
+
+echo "Install kubectl"
+wget --quiet --output-document=/tmp/Bluemix_CLI/bin/kubectl  https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

Comment:
We should note that this demo will only work on a Linux box or add a bit more checks to pull for OSX if detected.